modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -473,6 +475,24 @@ def obj.class
         end
       end
     end
+
+    context 'when submitted claim_date is in the future' do
+      let(:claim_date) { (Date.today + 1.day).to_s }
+
+      before do
+        stub_mpi
+      end
+
+
+      it 'creates the sidekick job' do
```

Comment:
We'd prob want a different test name here right?
Because it shouldn't be creating a job, but rather, raising an `InvalidFieldValue` exception.

Reply:
:(

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/7568/commits/37116890a4d979f8979d8dd3a0d2b56af8166bbb